apps.toml

```
@@ -1918,7 +1918,7 @@ url = "https://github.com/YunoHost-Apps/koel_ynh"
 [komga]
 added_date = 1674232499 # 2023/01/20
 category = "reading"
-level = 8
+level = 1
```

```suggestion
level = 8
```

fixed apparently by https://github.com/YunoHost-Apps/komga_ynh/pull/152